doc/specs/stdlib_stats_distribution_exponential.md

```diff
@@ -0,0 +1,243 @@
+---
+title: stats_distribution_exponential
+---
+
+# Statistical Distributions -- Exponential Distribution Module
+
+[TOC]
+
+## `rvs_expon` - exponential distribution random variates
```

Comment:
I was a bit surprised by the term `expon`. could it be `rvs_exp`? (probably a matter of taste).

Reply:
If most people like rvs_exp, I can certainly change it.

Reply:
FWIW, I vote for `rvs_exp`. Among other reasons, I think it's consistent with the Fortran intrinsic exponential function.

Reply:
I agree that `rvs_exp` is a good choice.